windows/src/developer/TIKE/compile/MergeKeyboardInfo.pas

```diff
@@ -775,6 +779,18 @@ procedure TMergeKeyboardInfo.CheckOrAddMinKeymanVersion;
     json.AddPair('minKeymanVersion', MinVersionString);
 end;
 
+//
+// Add helpPath, from commandline parameter
+//
+procedure TMergeKeyboardInfo.AddHelpPath;
+
```

Comment:
delete blank line

Reply:
chomp chomp still blank line to delete :)

Reply:
Hmm, I wonder if GitHub was caching the diff. 
When the procedure was named `AddHelpPath` it had the extra line.

It **should** be gone now...